cmd/policy_run.go

```diff
@@ -47,6 +48,10 @@ func init() {
 	flags := policyRunCmd.Flags()
 	flags.StringVar(&outputDir, "output-dir", "", "Generates a new file for each policy at the given dir with the output")
 	flags.BoolVar(&noResults, "no-results", false, "Do not show policies results")
+	flags.Bool("disable-fetch-check", false, "Disable checking if a respective fetch happened before running policies")
```

Comment:
`no-fetch-check`?
same goes for `disable-log-color` btw... we seem to have most of our "turn off" options as "no-option"

Reply:
Hmmm, I like "enable/disable" much more, because it's a verb. @roneli - what do you think?

Reply:
I would vote for consistency

Reply:
I think disable/enable is better, we should change all flags to disable/enable (but keep the old ones as well for backwards compatability eventually remove them)

Reply:
discussed in poll and changed back to disable/enable.